cmd/convox/templates/init/rails/docker-compose.yml

```diff
@@ -6,3 +6,5 @@ web:
   ports:
     - 80:4000
     - 443:4001
+  volumes:
```

Comment:
Also just a note that just because Docker is investing energy into it doesn't mean it's the right call. See overlay networks.